.pre-commit-config.yaml

```diff
@@ -16,7 +16,8 @@ repos:
     rev: v3.9.0
     hooks:
     -   id: reorder-python-imports
-        args: [--application-directories=src]
```

Comment:
this here is still needed otherwise pymovements is seen as external import

Reply:
yea -- fixed it, wanted to add you as reviewer after tests pass